core/eval.go

```diff
@@ -186,6 +186,15 @@ func evalSET(args []string) []byte {
 			if obj != nil {
 				return RESP_NIL;
 			}
+		case "GET":
+			obj := Get(key)
+
+			if obj == nil {
+				return RESP_NIL
+			}
+
+			// Return the RESP encoded value 
+			return Encode(obj.Value, false)
```

Comment:
If we just return the value from here then the flow would never reach the PUT statement, right? If that's the case the new value will never be set.

Reply:
Yes, actually, previously, I was also thinking like this only(I sent a screenshot also in the discussion channel), but after that, I realized that the Put statement is present in the last step and the value will go there automatically. Is it like that?

Or instead of that, we can do that, we can declare a tempObj and assign the obj value to it and then use the Put and the return statements.